dbtmetabase/metabase.py

```diff
@@ -131,6 +131,7 @@ def get_collection_items(
         self,
         uid: str,
         models: Sequence[str],
+        exclude_unverified: bool = False,
```

Comment:
This object is strictly a wrapper around Metabase API, don't add extra business logic (especially command-specific business logic) to it. You filter it at the point of invocation.